packages/bottender/src/router/index.ts

```diff
@@ -83,7 +83,7 @@ function text<C extends Client = any, E extends Event = any>(
   if (pattern instanceof RegExp) {
     return {
       predicate: (context: Context<C, E>) => {
-        const match = pattern.exec(context.event.text);
+        const match = pattern.exec(context.event.text ?? '');
```

Comment:
Is this correct?

Reply:
ref: https://github.com/tc39/proposal-nullish-coalescing

Reply:
oh, Why I wrote ?? instead of ||
Please focus on the correctness of the default value

Reply:
Looks good, empty string won't be matched.